pkg/trace/stats/aggregation_test.go

```diff
@@ -85,7 +85,7 @@ func TestNewAggregation(t *testing.T) {
 				Meta:    map[string]string{"span.kind": "client", "peer.service": "remote-service"},
 			},
 			false,
-			Aggregation{BucketsAggregationKey: BucketsAggregationKey{Service: "a", SpanKind: "client"}},
+			Aggregation{BucketsAggregationKey: BucketsAggregationKey{Service: "a", SpanKind: "client", IsParentRoot: true}},
```

Comment:
nit: I wouldn't expect a client span to be a parent root for the trace. Could we use a more realistic example?

Reply:
We probably don't need to update all of these other test cases.